src/app/charts/core/files/secrets/tpl/spring-config-saml.properties

```diff
@@ -0,0 +1,43 @@
+{{- $samlconfig := (include "arkcase.saml" $) -}}
+{{- if $samlconfig -}}
+  {{- $conf := get (fromYaml $samlconfig) "saml" -}}
+  {{- if $conf -}}
+# NOTE: we should further externalize all SAML-related Spring configuration properties
+# used in $HOME/.arkcase/acm/spring-security/spring-security-config-saml.xml
+
+# SECTION 1: configuration parameters for metadata generator
+
+# globally unique name for a SAML entity
+saml.metadataGenerator.entityId={{ $conf.entityId }}
+# base URL used in generated metadata to form service endpoints, such as single-logout or assertion-consumer service
+# it should be the URL as IdP sees it (usually a reverse-proxy address)
+saml.metadataGenerator.entityBaseURL=https://{{ $conf.arkcaseHost }}:{{ $conf.arkcasePort }}{{ $conf.arkcaseContextPath }}
+# federation metadata url (issuer's certificate must be added to ${user.home}/.arkcase/acm/sso/keystore.jks as trusted CA certificate)
+saml.federationMetadata.url={{ $conf.identityProviderUrl }}
+# federation metadata backup file location
+saml.federationMetadata.backupFile=${user.home}/.arkcase/acm/sso/idp.xml
+
+saml.keystore.file=${JAVA_KEYSTORE}
+saml.keystore.password=${JAVA_KEYSTORE_PASS}
+saml.keystore.defaultKeyName=${ACME_KEY_ALIAS}
+saml.keystore.defaultKeyPassword=${JAVA_KEYSTORE_KEY_PASS}
+
+# SECTION 2: configuration parameters for load balancer context provider
+
+# incoming endpoint protocol (as IdP sees it on reverse proxy)
+saml.contextProvider.scheme=https
+# incoming endpoint server name (as IdP sees it on reverse proxy)
+saml.contextProvider.serverName={{ $conf.arkcaseHost }}
+saml.contextProvider.serverPort={{ $conf.arkcasePort }}
+# incoming endpoint context path (as IdP sees it on reverse proxy)
+# must start with a forward slash!
+saml.contextProvider.contextPath={{ $conf.arkcaseContextPath }}
+
+# configure same as 'ldapConfig.userDomain' and 'ldapConfig.userPrefix' properties only when Azure SAML provider is used
+saml.userDomain=
+saml.userPrefix=
```

Comment:
I don't remember the issue with the Azure SAML provider.  With SAML, we expect to get the ArkCase user ID directly, since we use the ADFS claims mapping system to convert the incoming user ID to the expected ArkCase user ID.  Maybe with this Azure SAML provider, we couldn't use ADFS.  I looked at this source code, and it just adds the prefix if it's not already there, and sets the suffix if it's not already there.  Maybe I should change the comment.